Yoast/YoastSEO.js
# Add code tags to filter and add specs

## Summary

This PR can be summarized in the following changelog entry: Adds a filter that avoids text inside `<code>` tags to be included in the SEO and content assessments.

Please note: when using inline `<code>` tags, this will result in the marker not to mark text correctly. This unexpected behavior will be fixed in a future re-structuring of YoastSEO.js. For the moment, it has been decided to tolerate this behavior.

## Test instructions

This PR can be tested by following these steps:

- Use the browserified example. Don't forget to run `grunt build:js`.
- Write some text inside `<code>` tags.
- Make sure this text does not influence the scores in the SEO and content assessments.

Fixes #1245
Fixes #1051
